src/mem/minimal.cc

```diff
@@ -80,6 +80,12 @@ memAllocRigid(const size_t netSize)
     return xmalloc(netSize);
 }
 
+void
+memFreeRigid(void *buf, size_t)
```

Comment:
To pedantically improve const-correctness and for consistency with other similar use cases in this file (and a significant portion of new code):

```suggestion
memFreeRigid(void * const buf, size_t)
```